fn_grpc_interface/fn_grpc_interface/util/config.py

```diff
@@ -15,13 +15,13 @@ def config_section_data():
     config_data = u"""[fn_grpc]
 #interface_dir = <<path to interface buffer pb2 files parent directory>>
 #<<package_name=communication_type,secure connection type,certificate_path or google API token>>
-#helloword=unary,None,None
+helloword=unary,None,None
```

Comment:
I would keep this commented as we aren't expecting to have helloworld run out of the box.